memberships/models.py

```diff
@@ -122,6 +122,14 @@ class Member(models.Model):
         null=True,
         verbose_name="Membership renewal date",
     )
+    email_verified = models.BooleanField(
+        default=False,
+        verbose_name="Email verified",
+    )
+    email_verified = models.BooleanField(
+        default=False,
+        verbose_name="Email verified",
+    )
```

Comment:
Why is this repeated?

Reply:
Must've been a mistake when rebasing - thanks!